### ISSUE TITLE
Explain -JEPSG:n syntax in the cookbook

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#5767]
GMT started to support -JEPSG:n syntax since GMT 6. However, this new feature is only explained in the gmt manpage (https://docs.generic-mapping-tools.org/latest/gmt.html#the-j-option), not in the cookbook (https://docs.generic-mapping-tools.org/latest/cookbook/options.html#option-j).

This feature should be better documented in the cookbook.

**Comment by anbj** [comment]
Yes.

Suggestions:
* How set ``GDAL_DATA`` environment variable
* Add EPSG-codes (where applicable) to https://docs.generic-mapping-tools.org/latest/cookbook/options.html#projections-specifications

**Comment by joa-quim** [comment]
We don't need to set ``GDAL_DATA``. The directory is found automatically.

**Comment by anbj** [comment]
Ok, maybe reword or add to the paragraph

```
Using EPSG codes is also possible (but need the setting of the GDAL_DATA environment variable to point to the GDAL’s data sub-directory). For example -JEPSG:4326 sets the WGS-84 system.
```

I interpreted this as a need to manually add the path to ``GDAL_DATA``.

**Comment by joa-quim** [comment]
Yes, you are right. Needs rephrasing. That sentence was written before the the magic guessing ability was added to proj4 (``LIB_PROJ``). Regarding ``GDAL_DATA`` now you made me doubt but I'm not sure how much we use of it. The EPSG codes are no longer in ``GDAL_DATA`` but instead in ``LIB_PROJ`` (which is not needed because it's found automatically ``$GMT_SHARE/proj``).

**Comment by anbj** [comment]
Ah, OK. So changes indeed.

My ``GDAL_DATA`` environment variable is empty (pr. default), so I guess it's not used for much(?)

**Comment by joa-quim** [comment]
> Or am I misunderstanding?

No, you are on linux 😈 

On Windows I added the facility to proj4 to auto-detect the ``LIB_PROJ`` when it sits at ``bin/../share/proj``. This was later expanded to *unix and later again it searches not for ``bin`` but for directories with respect to the ``libproj.so`` (or whatever that lib is called in unix). [See ](https://github.com/OSGeo/PROJ/issues/2830)

I'm assuming that it also works on *nix but never tried.

**Comment by anbj** [comment]
Ok!

**Comment by GillesDuvert** [comment]
**Useful to be documented more**: programs using libproj on Windows (made with MINGW) should copy (in the installer script) the `msys64/share/proj` content to `"where_are_my_application_dlls"/../share/proj`

**Comment by joa-quim** [comment]
libproj built with MinGW will not work with the official GMT. For it to work the entire GMT must be build with MinGW. 
But why complicate? GMT comes with a nearly full proj installation.  Only the command line tools are not included.

**Comment by GillesDuvert** [comment]
Oh, sorry, this is probably the wrong place to pinpoint this, but I found this page after querying the web about the error: "Cannot find proj.db" and it was very useful to solve my problem, so it was in the hope it could help others.
My context is probably specific, but in [GDL](https://github.com/gnudatalanguage/gdl)  (the free clone of IDL) we use the proj library to handle projections, and we use the libproj only, not GMT, even if in many installs `libproj` is present because GMT is installed.

**Comment by joa-quim** [comment]
Interesting, I had never heard of GDL. Thanks for the pointer. 

I think that proj.db not found error cannot happen (cross fingers) with an official GMT installation.

**Comment by GillesDuvert** [comment]
> Interesting, I had never heard of GDL. Thanks for the pointer.
> 
recent versions permit now to make pretty maps... with projections unknown to IDL :smile
![image](https://user-images.githubusercontent.com/33936193/183687012-4cb746e5-846b-4ee1-89b5-046b38acf975.png)

> I think that proj.db not found error cannot happen (cross fingers) with an official GMT installation.
most probably no, indeed.

**Comment by joa-quim** [comment]
Nice. Could you try it with GMT?  As long as you don’t ask for a frame there are good chances that it works too.

**Comment by GillesDuvert** [comment]
Sorry, I would not know how to do that with GMT!... All graphics, shapefiles, contour cutting & filling, grids etc were done by GDL. Only the projection transformation is activated through a setup of PROJ, in this case it is just with options "+proj=bertin1953   +R=1.0000000".
The plot is extracted from the example procedure [here](https://github.com/gnudatalanguage/gdl/blob/master/testsuite/interactive_tests/test_map.pro), (example 8)

**Comment by joa-quim** [comment]
I think it would be something like (don’t remember how to set ocean land colors), but have no computer around and never mind. It was just curiosity. 

coast -Rd -W0.5 -Bg -J”+proj=bertin1953+R=1.0000000+width=10c” -png
